config.json

```diff
@@ -97,6 +97,7 @@
 		}
 	},
 	"broadcasts": {
+		"on": true,
```

Comment:
having this property named as `active` would be more descriptive. What do you think?